loopchain/peer/consensus_siever.py

```diff
@@ -126,8 +126,8 @@ def __get_complaint_votes(self):
         util.logger.debug(f"-------------------consensus-------------------")
         async with self.__lock:
             if self._block_manager.epoch.leader_id != ChannelProperty().peer_id:
-                util.logger.warning(f"This peer is not leader. epoch leader={self._block_manager.epoch.leader_id}")
-                return
```

Comment:
why this return line removed?

Reply:
When the term transitions, the leader is not the leader but must make the last block. When leader make last block of Term, that leader is no more leader but must make the last block.